src/Microsoft.AspNetCore.Diagnostics.Abstractions/IExceptionHandlerFeature.cs

```diff
@@ -8,5 +8,7 @@ namespace Microsoft.AspNetCore.Diagnostics
     public interface IExceptionHandlerFeature
     {
         Exception Error { get; }
+
+        string Path { get; }
```

Comment:
> but is it an acceptable break

I have no idea. @Eilon usually doesn't like any sort of breaking changes in minor releases.

An alternate way to fix this without breaking would be to introduce a new interface and add the property on there.